_storage/amazon-s3-deploy.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# Configure a hub on Amazon S3
```

Comment:
These instructions read fine to me -- I'll need to test them once I've synced with Jesse on the AMI status

Reply:
Cool. Thanks.